packages/server/src/automations/steps/queryRows.ts

```diff
@@ -122,9 +122,10 @@ export async function run({
     sortType =
       fieldType === FieldType.NUMBER ? FieldType.NUMBER : FieldType.STRING
   }
+  // when passing the tableId in the Ctx it needs to be decoded
   const ctx = buildCtx(appId, null, {
     params: {
-      tableId,
+      tableId: decodeURIComponent(tableId),
```

Comment:
I'm confused as to why spaces in the _name_ of the table would affect the- ohhhh the ID for external tables encodes the name within it doesn't it?

Reply:
Yeap exactly this - external tables the ID is built from the table name today.